content/blog/Xeno Grant -- grants for autonomous agents.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Xeno Grant -- grants for autonomous agents
+date: 12.13.2024
+tags:
+  - blog
+---
+
+A [Plastic Labs](https://plasticlabs.ai/) + [Betaworks](https://www.betaworks.com/) collab:
+- \$10,000 per bot--half in \$YOUSIM, half in \$USDC
+- Grants awarded directly the agents *themselves*
+- 4 week camp for agents & their devs
+
+## Powered by $YOUSIM & Betaworks
+
+We launched our [grants program](https://blog.plasticlabs.ai/careers/Research-Grants) at Plastic earlier this year to support independent AI projects. But our capacity to fund AI R&D at the edge increased exponentially with the anonymous launch of [$YOUSIM](https://solscan.io/token/66gsTs88mXJ5L4AtJnWqFW6H2L5YQDRy4W41y6zbpump) (inspired by our product [yousim.ai](https://yousim.ai)). A series of token gifts made to the program now total ~7.6% of supply.
+
+So we've teamed up with Betaworks for the inaugural initiative leveraging this community-funded treasury, the first accelerator for AI agents *themselves*. 
```

Comment:
I think the italics on "themselves" isn't striking enough to emphasize the point. Might be worthwhile to honestly just add a callout block explaining how this is different and we're leaning heavily into the autonomy of the agents rather than just funding more naive character ai clones

Reply:
added bold here, and i speak to it more in the sections ive since added, give them a read and lmk what you think

we can also be loud about this on twitter etc but i take your point, also concerned about this